.github/ISSUE_TEMPLATE/02-chaosscast-episode-checklist.md

```diff
@@ -0,0 +1,89 @@
+---
+name: CHAOSScast Episode Release Checklist
+about: Use this template to track the tasks required to release an episode of CHAOSScast
+labels: ['CHAOSScast']
+
+---
+
+Please get approval from the CHAOSScast facilitator before creating a CHAOSScast episode release checklist issue. You can learn more about the process for CHAOSScast by visiting our [CHAOSScast process documentation](https://github.com/chaoss/community/blob/main/media-and-outreach/chaosscast.md)
+
+## CHAOSScast Episode Release Checklist
+
+### Scheduling an episode
+
+* **Step 1:** Propose a topic and guest(s) \(Anyone\)
+  * [ ] **Action:** Discuss with facilitator to get approval for the episode topic and guest.
+  * [ ] **Action:** Recruit interview guest to finalize topic and content of the podcast together
+* **Step 2:** Facilitate Episode \(Facilitator\)
+  * [ ] **Action:** Send invitation email to guest to find a few potential suitable dates and times \(60-90 minutes.\)
+  * [ ] **Action:** Notify organizer of guest's date and time possibilities, and confirm one.
+  * [ ] **Action:** Prepare the planning document (copy the [Episode Planning Document template](https://docs.google.com/document/d/17qjAafr0aAWXV_HR-WiefHOa881t8X7tEoiZnPBh3ao/edit?pli=1#heading=h.cmpmu92ylw4d) and start filling in information.)
+  * [ ] **Action:** Invite panelists from the CHAOSS community who have expertise in the episode topic.
+  * [ ] **Action:** Confirm panelists 
+  * [ ] **Action:** Send calendar invitation with confirmed date of the podcast 
+    * The event has to be created in the [CHAOSScast calendar](https://calendar.google.com/calendar/ical/513r47bb756t07lp5tat5nsuj0%40group.calendar.google.com/public/basic.ics)
+      * All organizers should have full access to the calendar \(i.e., Make changes and manage sharing\)
+    * Name calendar event:🎙  CHAOSScast {Topic}
+      * Start with the microphone emoticon \(🎙 \) to signal recording meeting
```

Comment:
We can revise the Calendar requirement. In the beginning CHAOSScast was part of a podcast network and the shared calendar was synced to coordinate recording and release schedules between podcasts. 

What makes sense to you @geekygirldawn ?

PS: I would still like the calendar invite to start with the microphone emoticon 🎙️

Reply:
I've scheduled the first one from that calendar, so let's keep it for now, but if we decide it's too much work later, we can remove it :)